<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>876</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>631</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="robots_layout">
     <item row="1" column="0">
      <widget class="QLabel" name="robot_label">
       <property name="text">
        <string>Robots</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="grade_btn">
       <property name="text">
        <string>GRADE</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="login_btn">
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="Line" name="robot_line_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="robot_line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>122</y>
      <width>661</width>
      <height>701</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>657</width>
       <height>697</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="quicklinks_layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="quicklinks_label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Quicklinks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="quicklinks_line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="post_checkbox">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>POST</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="post_grid">
          <item row="0" column="1">
           <widget class="QLabel" name="post_home_link">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="post_mkt200_checkbox">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>MKT200</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QGridLayout" name="post_mkt200_grid">
            <item row="1" column="1">
             <widget class="QLabel" name="post_mkt200_announcements">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Announcements</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="post_mkt200_grading_center">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Grading center</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="post_mkt200_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="post_bus311_checkbox">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>BUS311</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="post_announcements_link">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Announcements</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="quicklinks_verticalspacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
